package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "nypl-discovery",
```

Comment:
hmmm I don't think so. It's the repo's name, not the app name. And the way that it's going, the app name could change. AGAIN.